bigtable_automation/gcf/custom_dc/local.yaml

```diff
@@ -1,7 +1,8 @@
-projectID: "google.com:datcom-store-dev"
+projectID: "datcom-mixer-encode"
 instance: "prophet-test"
 cluster: "prophet-test-c1"
 dataflowTemplate: "gs://datcom-dataflow-templates/templates/flex/csv_to_bt.json"
 tempLocation: "gs://datcom-store-dev-resources/dataflow/tmp"
-controllerTriggerTopic: "projects/datcom-204919/topics/custom-import-notification-test"
-bucket: "automation_control_test"
+controllerTriggerTopic: "projects/datcom-204919/topics/private-import-notification-dev"
+bucket: "datcom-mixer-encode-resources"
+dataDirectory: "demo/data/"
```

Comment:
the data directory "demo" is set by the user, this may not work if there are other folders?